WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -246,23 +255,25 @@ def post_results
         if top_three.empty?
           return render html: "<div class='container'><div class='alert alert-warning'>Nobody competed in event: #{event.id}</div></div>".html_safe
         else
-          first_result = top_three.first
+          results_by_place = top_three.group_by(&:pos)
+          winners = results_by_place[1]
 
-          title = "#{first_result.personName} wins #{comp.name}, in #{comp.cityName}, #{comp.country.name}"
+          title = "#{people_to_sentence(winners, link: false)} #{winners.length  > 1 ? "win" : "wins"} " \
```

Comment:
Nit: I think there are two spaces before the `>` here.

Reply:
Good eye! Travis didn't hesitate to point this out as well ^^
```

app/controllers/competitions_controller.rb:261:80: C: Layout/ExtraSpacing: Unnecessary spacing detected.
          title = "#{people_to_sentence(winners, link: false)} #{winners.length  > 1 ? "win" : "wins"} " \
                                                                               ^
```